### ISSUE TITLE
RawPhenotypes needs to be updated.

### ISSUE THREAD

**Issue by laceysanderson** [UofS-Pulse-Binfo/KnowPulse#14]
UofS-Pulse-Binfo/rawphenotypes#13 has been approved 👍  Thus the portal needs to be updated.

Putting the upload on hold until @reynoldtan fixes UofS-Pulse-Binfo/rawphenotypes#15 and UofS-Pulse-Binfo/rawphenotypes#8.

**Comment by laceysanderson** [comment]
Also from https://github.com/UofS-Pulse-Binfo/rawphenotypes/pull/30

**Comment by laceysanderson** [comment]
Waiting on #31

**Comment by laceysanderson** [comment]
Also depends on https://github.com/UofS-Pulse-Binfo/kp_nodes/pull/7

**Comment by laceysanderson** [comment]
Currently waiting on UofS-Pulse-Binfo/rawphenotypes#47

**Comment by laceysanderson** [comment]
Done
